src/plugins/intel_gpu/src/plugin/ops/transpose.cpp

```diff
@@ -36,11 +36,19 @@ static void CreateTransposeOp(Program& p, const std::shared_ptr<ngraph::op::v1::
                ngraph::is_type<ngraph::op::v8::I420toBGR>(node);
     };
 
+    auto is_convert = [](const std::shared_ptr<ov::Node> &node) {
+        return ngraph::is_type<ngraph::op::v0::Convert>(node);
+    };
+
     // Handle Transpose operation related to ConvertColor operation:
     // In case of ConvertColor operation we have NHWC (byxf) input format which should be converted to
     // NCHW (bfyx) by this Permute, so we replace Permute with Reorder (to bfyx) primitve
     auto input = op->input(0).get_source_output().get_node_shared_ptr();
-    if (is_convert_color_type(input) && order == std::vector<uint16_t>{0, 3, 1, 2}) {
+    auto input2 = input;
+    if (is_convert(input)) {
+        input2 = op->get_input_node_shared_ptr(0)->input(0).get_source_output().get_node_shared_ptr();
+    }
+    if ((is_convert_color_type(input) || is_convert_color_type(input2)) && order == std::vector<uint16_t>{0, 3, 1, 2}) {
```

Comment:
How about changing `is_convert` lambda function to return boolean type with checking `is_convert_color_type` as well? Because if `is_convert` is false, duplicate code for the same parameter `input`.

Reply:
Agree. I think it would be better to check "is_convert" and check input node of convert inside the is_convert_color_type.

Reply:
how about like this 
bool is_convert_color_type = [] () {
    if (input is convert) {
       return (is_convert_color(input->get_input());
    }
    // is_convert_color_type impl
}

Reply:
Fixed